stack-scholar/recipes/role-consumer.rb

```diff
@@ -0,0 +1,12 @@
+include_recipe 'supervisor'
+include_recipe 'stack-easybib::role-phpapp'
+
+unless node['haproxy']['websocket_layers'].nil?
+  if has_role?(get_instance_roles, node['haproxy']['websocket_layers'].keys.first)
+    node.set['nodejs']['version'] = '4.2.6'
```

Comment:
@till Not sure about the use case here, but maybe this should be `override`? (https://docs.chef.io/attributes.html)